conf/messages.cy

```diff
@@ -66,6 +67,10 @@ contact.heading=Yr hyn i’w wneud os ydych yn cael anhawster wrth ddefnyddio’
 contact.extra-support.part1='Gallwch '
 contact.extra-support.part2=gysylltu â CThEM os oes angen cymorth ychwanegol
 contact.extra-support.part3=' arnoch i lenwi ffurflenni, neu i gael fformat gwahanol, megis print bras, recordiad sain neu Braille.'
+contact.extra-support-voa.part1='Gallwch '
+contact.extra-support-voa.part2=cysylltwch â VOA am gymorth ychwanegol os oes angen help arnoch i lenwi ffurflenni neu gael fformat gwahanol, fel print bras, recordiad sain neu Braille.
+contact.extra-support-voa.part3=Ffôn: 03000 501 501
+contact.extra-support-voa.part4=Oriau agor: Dydd Llun i Ddydd Gwener: 9:30yb i 2:30yp
```

Comment:
I'm not sure if this is the correct way to express pm and am in welsh?

Reply:
@oscarduignan They never provided a welsh translation, so what I have is a pure google translation. I guess the best thing to do here is to ask them directly?